server/lib/genome/importers/api_importers/pharos/api_client.rb

```diff
@@ -27,25 +27,35 @@ def headers(_context)
         end
       end
 
-      Schema = GraphQL::Client.load_schema(HTTP)
-      Client = GraphQL::Client.new(schema: Schema, execute: HTTP)
-    end
+      begin
+        Schema = GraphQL::Client.load_schema(HTTP)
+        Client = GraphQL::Client.new(schema: Schema, execute: HTTP)
 
-    Query = PharosApi::Client.parse <<-GRAPHQL
-      query($filter: IFilter, $skip: Int, $top: Int) {
-        targets(filter: $filter) {
-          targets(skip: $skip, top: $top) {
-            uniprot
-            name
-            sym
-            fam
-            preferredSymbol
+        Query = PharosApi::Client.parse <<-GRAPHQL
+          query($filter: IFilter, $skip: Int, $top: Int) {
+            targets(filter: $filter) {
+              targets(skip: $skip, top: $top) {
+                uniprot
+                name
+                sym
+                fam
+                preferredSymbol
+              }
+            }
           }
-        }
-      }
-    GRAPHQL
+        GRAPHQL
+      rescue KeyError => e
+        puts "Error initializing Pharos GraphQL client: #{e.message}"
```

Comment:
@acoffman is there a more helpful way to log this for the production environment?

Reply:
`Rails.logger.warn` or `Rails.logger.error` would be more canonical ways to log this, and then if you end up using logstash or cloudwatch or something it'll go to the right place instead of just stdout/stderr.